EIPS/eip-6956.md

```diff
@@ -0,0 +1,665 @@
+
+- **Selling a house with a lease:** A lease contract puts a lien on an ASSET's anchored NFT. The old owner removes the lock, the new owner buys and refinances the house. Transfer of NFT will also transfer the obligations and benefits of the lien to the new owner. As a lien-interface, the proposed EIP can for example be extended with [ERC-5604](eip-5604.md)
+
+- **Buying a brand new car with downpayment:** A buyer configures a car and provides a downpayment, for a car that will have an ANCHOR. As long as the car is not produced, the NFT can float and be traded on NFT market places. The owner of the NFT at time of delivery of the ASSET has the the permission to pick up the car and the obligation to pay full price.
+
+- **Buying a barrel of oil by forward transaction:** A buyer buys an oil option on a forward contract for one barrel of oil (ASSET). On maturity date the buyer has the obligation to pick up the oil.
+
+The use case matrix below shows which extensions and settings must (additionally to `IERC6956`!) be implemented for the example use-cases together with relevant configurations.
+
+Note that for `Lockable` listed in the table below, the proposed EIP can be extended with any Lock- or Lien-Mechanism known to extend for ERC-721. Suitable extensions to achieve `Lockable` are for example [ERC-5058](eip-5058.md) or [ERC-5753](eip-5753.md). We recommend to verify whether a token is locked in the `_beforeTokenTransfer()`-hook, as this is called from `safeTransferFrom()` as well as `transferAnchor()`, hence suitable to block "standard" ERC-721 transfers as well as the proposed attestation-based transfers.
+
+
+| Use Case | approveAuthorization | burnAuthorization | `IERC6956Floatable` | `IERC6956AttestationLimited` | Lockable |
+|---------------|---|---|---|---|---|
+| **Managing Posession** |
+| Token gating  | ASSET | ANY | incompatible | - | - |
+| Digital twin  | ASSET | ANY | incompatible | - | - |
+| Scarce digital twin | ASSET | ANY | incompatible | required | - |
+| Lendable digital twin         | OWNER_AND_ASSET | ASSET | required | - | - |
+| **Managing Ownership** |
+| Securing ownership from theft   | OWNER or OWNER_AND_ASSET | ANY | optional | - | required |
+| Selling an house with a mortage  | ASSET  or OWNER_AND_ASSET | ANY | optional | optional | required |
+| Selling a house with a lease | ASSET or OWNER_AND_ASSET | ANY | optional | optional | required |
+| Buying a brand new car with downpayment | ASSET or OWNER_AND_ASSET | ANY | optional | optional | required |
+| Buying a barrel of oil by forward transaction | ASSET or OWNER_AND_ASSET | ANY | optional | optional | required |
+
+
+Legend:
+
+- required ... we don't see a way how to implement the use-case without it
+- incompatible ... this MUSTN'T be implemented, as it is a security risk for the use-case
+- optional ... this MAY optionally be implemented
+
+
+## Backwards Compatibility
+
+---- NEEDS FURTHER DISCUSSION ----
```

Comment:
If you use a comment for these notes, the linting tools will make sure you remove them before moving into review:

```suggestion
<!-- NEEDS FURTHER DISCUSSION -->
```

Reply:
Done in https://github.com/ethereum/EIPs/pull/6956/commits/1713073d9c204829c446afffa7afd4fbd7754bd6

Thx for the hint with linter!